blazor/spreadsheet/cellrange.md

```diff
@@ -0,0 +1,124 @@
+
+Cell formatting options include:
+
+* **Bold** - Applies a heavier font weight to make the text stand out in the Spreadsheet.
+
+* **Italic** - Slants the text to give it a distinct look, often used for emphasis or to highlight differences.
+
+* **Underline** - Adds a line below the text, commonly used for emphasis or to indicate hyperlinks.
+
+* **Strikethrough** - Draws a line through the text, often used to show completed tasks or outdated information.
+
+* **Font Family** - Changes the typeface of the text (e.g., Arial, Calibri, Times New Roman) to enhance readability or visual appeal.
+
+* **Font Size** - Adjusts the size of the text to create visual hierarchy or improve readability in the Spreadsheet.
+
+* **Font Color** - Changes the color of the text to improve visual hierarchy or to organize information using color codes.
+
+* **Fill Color** - Adds color to the cell background to visually organize data or highlight important information.
+
+* **Horizontal Alignment** - Controls the position of text from left to right within a cell. Options include:
+    * **Left** - Default for text
+    * **Center** - Useful for headings
+    * **Right** - Default for numbers
+
+* **Vertical Alignment** - Controls the position of text from top to bottom within a cell. Options include:
+    * **Top** – Aligns content to the top of the cell
+    * **Middle** – Centers content vertically
+    * **Bottom** – Default alignment
+
+* **Wrap Text** - Displays long content on multiple lines within a single cell, preventing it from overflowing into adjacent cells.
+
+Cell formatting can be applied to or removed from a cell or range of cells by using the formatting options available in the ribbon toolbar under the Home tab.
+
+## Autofill
+
+Autofill is used to fill the cells with data based on adjacent cells. It also follows a pattern from adjacent cells if available. There is no need to enter the repeated data manually. The [AllowAutofill](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Spreadsheet.SfSpreadsheet.html#Syncfusion_Blazor_Spreadsheet_SfSpreadsheet_AllowAutofill) property can be used to enable/disable the autofill support.
+
+> * The default value of `AllowAutofill` is **true**.
+
+This can be done in one of the following ways:
+
+* Drag and drop the cell using fill handle element.
+* Use the `AutofillAsync()` method programmatically.
+
+The available parameters in AutofillAsync() method are,
+
+| Parameter | Type | Description |
+| -- | -- | -- |
+| fillRange | string | Specifies the fill range. |
+| dataRange | string | Specifies the data range. |
+| direction | string | Specifies the direction ("Up", "Right", "Down", "Left") to be filled. |
+
+### Implementing Autofill Programmatically
+
+{% tabs %}
+{% highlight razor tabtitle="Index.razor" %}
+
+@using Syncfusion.Blazor.Spreadsheet
+
+<button @onclick="AutoFillRangeHandler">Autofill</button>
```

Comment:
Code snippet Click handler name was incorrect, change it as **AutofillRangeHandler** instead of **AutoFillRangeHandler**